orchagent/portsorch.cpp

```diff
@@ -537,23 +537,48 @@ void PortsOrch::doLagTask(Consumer &consumer)
             Port lag, port;
             getPort(lag_alias, lag);
             getPort(port_alias, port);
+            string admin_status;
 
             if (op == SET_COMMAND)
             {
-                /* Duplicate entry */
-                if (lag.m_members.find(port_alias) != lag.m_members.end())
+                for (auto i : kfvFieldsValues(t))
```

Comment:
kfvFieldsValues return a vector instead of a map. so here we use for loop for now. the data structure could be optimized in the future.